plugins/source/azure/resources/services/cognitiveservices/account_skus_models.go

```diff
@@ -0,0 +1,52 @@
+package cognitiveservices
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices"
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func accountModels() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_cognitiveservices_account_models",
+		Resolver:             fetchAccountModels,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices@v1.3.0#AccountModel",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_cognitiveservices_account_models", client.Namespacemicrosoft_cognitiveservices),
+		Transform:            transformers.TransformWithStruct(&armcognitiveservices.AccountModel{}),
```

Comment:
These don't have an ID. In `azure_compute_skus` we calculate a hash. We could do this here too.
Also the models list method is not shown in the REST API docs so I'm linking to the `pkg.go.dev` docs in the description